integrations/datadog/client.py

```diff
@@ -233,6 +297,176 @@ async def get_slo_history(
             )
             return {}
 
+    async def get_tags(self) -> dict[str, Any]:
+        url = f"{self.api_url}/api/v1/tags/hosts"
```

Comment:
why /hosts if we said `serviceMetrics` I am a bit confused

Reply:
tags are picked from hosts in the datadog agent. The agent will pick all tags from a host while sending metrics, tracces, logs, e.t.c